views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/uiElements.js

```diff
@@ -404,12 +404,56 @@ define([
         return mediaStimulus;
     }
 
+    /**
+     * Creates a countdown timer as a pie chart
+     * @param {Object} config
+     * @param {Object} config.$container - jQuery element that the countdown timer will be appended to
+     * @param {Number} config.delayInSeconds - delay in seconds
+     */
+    function countdownPieChartFactory(config) {
+        var countdownPieChart;
+        var $container   = config.$container;
+        var delay = config.delayInSeconds - 1;
+        var $countdownPieChart = $(
+                '<div class="countdown-pie-container countdown-pie-animated">' +
```

Comment:
This indentation doesn't seem correct, can you check it please.
I sugest to run linter in all modified files.

Reply:
This should use a Handlebars template instead. A direct build of HTML breaks the MVC pattern and is not allowed, unless strong reason.

Reply:
It seems PCI is an exception to the rule, as importing Handlebars template could be tricky.

Reply:
Ok. No problem.